android/modules/ui/src/java/ti/modules/titanium/ui/android/FabProxy.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * TiDev Titanium Mobile
+ * Copyright TiDev, Inc. 04/07/2022-Present
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+package ti.modules.titanium.ui.android;
+
+import android.app.Activity;
+
+import org.appcelerator.kroll.annotations.Kroll;
+import org.appcelerator.titanium.proxy.TiViewProxy;
+import org.appcelerator.titanium.view.TiUIView;
+
+import ti.modules.titanium.ui.widget.TiUIFab;
+
+@Kroll.proxy(creatableInModule = AndroidModule.class)
+public class FabProxy extends TiViewProxy
```

Comment:
We should handle it as a `style` property in the existing `Button` proxy or call it `FloatingActionButton`

Reply:
I think I'll rename it to `FloatingActionButton` since its an own element in Android too it shouldn't be mixed with the button. Otherwise we have many non-parity properties

Reply:
done :)